addons/60fps/userscript.js

```diff
@@ -18,6 +18,7 @@ export default async function ({ addon, global, console }) {
     let mode = false;
     const flagListener = (e) => {
       if (altPressedRecently) {
+        e.cancelBubble = true;
```

Comment:
why do you need this line?

Reply:
This line will not allow the project to restart when the flag is pressed.

Reply:
oh i was just confused as it doesn't look like it's related to this addon